app/src/main/java/com/duycuong/weather/ui/screen/main/current_weather/CurrentWeatherViewModel.java

```diff
@@ -0,0 +1,126 @@
+package com.duycuong.weather.ui.screen.main.current_weather;
+
+import android.content.Context;
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+import android.location.Location;
+import android.widget.Toast;
+
+import com.duycuong.weather.BR;
+import com.duycuong.weather.R;
+import com.duycuong.weather.data.model.CurrentWeather;
+import com.duycuong.weather.data.model.CurrentlyResponse;
+import com.duycuong.weather.data.source.WeatherDataSource;
+import com.duycuong.weather.data.source.WeatherRepository;
+import com.duycuong.weather.data.source.remote.WeatherRemoteDataSource;
+import com.duycuong.weather.utils.Constant;
+import com.duycuong.weather.utils.ConvertTime;
+import com.duycuong.weather.widget.dialog.progress.DialogManager;
+
+/**
+ * Created by DuyCương on 08/02/2018.
+ */
+
+public class CurrentWeatherViewModel extends BaseObservable {
+    private CurrentWeather mCurrentWeather;
+    private String mTime;
+    private int mIcon;
+    private Context mContext;
+    private DialogManager mDialogManager;
+    private WeatherRepository mWeatherRepository;
+
+    public CurrentWeatherViewModel(Context context) {
+        mContext = context;
+        mDialogManager = new DialogManager(context);
+        mWeatherRepository = new WeatherRepository(new WeatherRemoteDataSource());
+        mDialogManager.showDialog();
+    }
+
+    @Bindable
+    public CurrentWeather getCurrentWeather() {
+        return mCurrentWeather;
+    }
+
+    public void setCurrentWeather(CurrentWeather currentWeather) {
+        mCurrentWeather = currentWeather;
+        notifyPropertyChanged(BR.currentWeather);
+    }
+
+    @Bindable
+    public String getCurrenTime() {
+        return mTime;
+    }
+
+    public void setCurrentTime(String time) {
+        mTime = time;
+        notifyPropertyChanged(BR.currenTime);
+    }
+
+    @Bindable
+    public int getIcon() {
+        return mIcon;
+    }
+
+    public void setIcon(int idIcon) {
+        mIcon = idIcon;
+        notifyPropertyChanged(BR.icon);
+    }
+
+    public void getTheCurrentWeather(Location location) {
+        mWeatherRepository.getWeatherCurrent(location,
+                new WeatherDataSource.Callback<CurrentlyResponse>() {
+            @Override
+            public void onStartLoading() {
+                //no_ops
+            }

```

Comment:
ERROR: 'method def rcurly' have incorrect indentation level 12, expected level should be one of the following: 16, 20.